src/Extensions/SiteTreeExtension.php

```diff
@@ -354,13 +431,9 @@ public function MetaTags(&$tagString)
             $socialMetaTags = $this->owner->updateSocialMetaTags($socialMetaTags);
         }
 
-        $tagString .= "\n" . implode("\n", $socialMetaTags);
-
-        $extraMeta = $this->owner->getSocialMetaValue('ExtraMeta');
-        if ($extraMeta) {
-            $tagString .= "\n" . $extraMeta;
```

Comment:
I would leave this here as the ExtraMeta data is a string containing a pre-rendered html element. Unless we introduce a breaking change and change the return value of `getDefaultSocialMetaExtraMeta()` and `getSocialMetaExtraMeta()`?

Reply:
This should be moved back to `MetaTags()`. I think the goal of this is to convert the usage vs holistically break things. The move to `MetaComponents` for `VirtualPage` was accepted in a minor or patch release so we shouldn't need to do breaking changes.